Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "IsPhantomCameraA", "type": "bool" },
+        { "name": "EaseTimeA", "type": "float32" },
+        { "name": "PriorityA", "type": "uint32" },
+        { "name": "NoneEffectiveBoostA", "type": "bool" },
+        { "name": "CameraB", "type": "object_reference" },
+        { "name": "CollisionB", "type": "object_reference" },
+        { "name": "SideTypeB", "type": "CameraCollisionBoardSpawner::SideTypeB" },
+        { "name": "IsPhantomCameraB", "type": "bool" },
+        { "name": "PriorityB", "type": "uint32" },
+        { "name": "EaseTimeB", "type": "float32" },
+        { "name": "NoneEffectiveBoostB", "type": "bool" }
+      ]
+    },
+    "EventVolumeCameraChangeSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" },
+        { "name": "Target", "type": "object_reference" },
+        { "name": "IsPhantomCamera", "type": "bool" },
+        { "name": "PriorityCategory", "type": "EventVolumeCameraChangeSpawner::PriorityCategory" },
+        { "name": "Priority", "type": "uint32" },
+        { "name": "EaseTimeEnter", "type": "float32" },
+        { "name": "EaseTimeLeave", "type": "float32" },
+        { "name": "ShapeType", "type": "EventVolumeCameraChangeSpawner::ShapeType" },
+        { "name": "IsStop", "type": "EventVolumeCameraChangeSpawner::IsStop" },
+        { "name": "IsEnableCollision", "type": "bool" },
+        { "name": "InterpolateType", "type": "EventVolumeCameraChangeSpawner::InterpolateType" },
+        { "name": "PhantomType", "type": "EventVolumeCameraChangeSpawner::PhantomType" },
+        { "name": "NoneEffectiveBoost", "type": "bool" }
+      ]
+    },
+    "ObjCameraFixSpawner": {
+      "parent": "ObjCameraSpawner",
+      "fields": [
+        { "name": "Fovy", "type": "float32" },
+        { "name": "ZRot", "type": "float32" },
+        { "name": "TargetPosition", "type": "vector3" }
+      ]
+    },
+    "ObjCameraObjectLookSpawner": {
+      "parent": "ObjCameraSpawner",
+      "fields": [
+        { "name": "Fovy", "type": "float32" },
+        { "name": "ZRot", "type": "float32" },
+        { "name": "TargetType", "type": "ObjCameraObjectLookSpawner::TargetType" },
+        { "name": "Target", "type": "object_reference" },
+        { "name": "TargetPositionFix", "type": "vector3" },
+        { "name": "TargetObject", "type": "object_reference" },
+        { "name": "Distance", "type": "float32" },
+        { "name": "TargetOffsetWorldX", "type": "float32" },
+        { "name": "TargetOffsetWorldY", "type": "float32" },
+        { "name": "TargetOffsetWorldZ", "type": "float32" },
+        { "name": "TargetOffsetRight", "type": "float32" },
+        { "name": "TargetOffsetUp", "type": "float32" },
+        { "name": "TargetOffsetFront", "type": "float32" }
+      ]
+    },
```

Comment:
Unused object. Parameters named based on in-game functionality and the object with the same name found in Sonic Unleashed/Generations. Additional help was used by comparing the set data parser function of this obejct and `PointCamera`. Parameter names from SU/Gens have been updated to match other camera objects in Sonic Colors.

More information on the object at:
https://hedgedocs.com/discoveries/sonic2010/objects/unused/#objectlookcamera